.env

```diff
@@ -0,0 +1,17 @@
+# Database - change to fit your database
+DB_HOST = 0.0.0.0
```

Comment:
Make sure to always add this file to git ignore. It should not be committed to the repository since it contains sensitive information (password etc)